watertap/property_models/multicomp_aq_sol_prop_pack.py

```diff
@@ -2274,9 +2270,14 @@ def calculate_scaling_factors(self):
 
         if self.is_property_constructed("ionic_strength_molal"):
             if iscale.get_scaling_factor(self.ionic_strength_molal) is None:
-                sf = min(
-                    iscale.get_scaling_factor(self.molality_phase_comp["Liq", j])
-                    for j in self.params.solute_set
+                sf = (
+                    sum(
+                        iscale.get_scaling_factor(self.molality_phase_comp["Liq", j])
+                        ** -1
+                        for j in self.params.solute_set
+                    )
+                    ** -1
+                    / 2
                 )
                 iscale.set_scaling_factor(self.ionic_strength_molal, sf)
```

Comment:
Just had a new finding:  it should probably be decided by `len()` in cases where there are more than 2 addends.

Reply:
nvm, I misread the eq.  Still approval.